oauth/o-auth.md

```diff
@@ -69,6 +69,14 @@ Once you click on ``Try it!`` you will see a modal pop up where you will have to
 Once you've successfully integrated your Google credentials with the connections section of Auth0 you can proceed with going to ``Applications`` and then type in ``React`` in the search bar. </br>
 <img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/Screen%20Shot%202018-10-09%20at%205.58.09%20PM.png"  width="500" height="500" /></br></br>
 
+#### Step 4 - Create a working log-out button such that the button is shown when the user is authenticated, providing an option to log-out successfully </br>
+Once the user has successfully login,they are rendered with the option of log-out button, the following code in the *app.js* file can be added to obtain so.</br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/pull/1/commits/82d3380590b9962370c2635905e27f2be7805677?short_path=bb1a3d8#diff-bb1a3d83f4321e7da9d790cce464c4f6"></br>
+
+We have to open *Auth.js* file in the sample downloaded folder and add logout method(it removes tokens and expiry time) as the following code snippet.In the following "logout" method, the auth token is removed from localStorage and redirect to a home page.</br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/pull/1/commits/8d50a070a6b1609a078a9035fd9e6fda885335eb?short_path=91cd898#diff-91cd898e94c40975ed3d2e1b9b287a9b"></br>
+
+
 ##### Common Questions
 -If you get an error about **registerServiceWorker.js** not being found, serviceWorker.js can be renamed to registerServiceWorker.js to match all the times it's being required in other files.
 -You may get an error saying that registerServiceWorker.js has **no default export**.  In registerServiceWorker.js, register(config) should be the default export if it isn't already: (line 23 or so) ```export default function register(config) {```
```

Comment:
I am very sorry, i have made the following commits incorrectly.  The following commit was for another PR.